plugins/scaffolder-backend/api-report.md

```diff
@@ -312,7 +312,10 @@ export const createPublishGitlabMergeRequestAction: (options: {
     sourcePath?: string | undefined;
     targetPath?: string | undefined;
     token?: string | undefined;
-    commitAction?: 'update' | 'delete' | 'create' | undefined;
+    commitAction?: 'auto' | 'update' | 'delete' | 'create' | undefined
+    /**
+     * @public @deprecated use import from \@backstage/plugin-scaffolder-backend-module-github instead
```

Comment:
Not sure that this should be in here?

Reply:
You should just be able to run `yarn build:api-reports` locally and it update the files for you automatically. The comments are never in the `api-report` so it looks like something strange happened, looks like it could have been a manual update that added this in here. 

Anyhoo, it doesn't matter, I can re-run and update and see if we can get this merged :tada:

Reply:
Oh what - apologies, you're 100% correct that you did the right thing, this looks to be a bug or something. I've never seen this happen in the api-reports.md before. /cc @Rugvip

Reply:
Think we managed to fix the issue with the parser here #25740

Reply:
resolved ?